src/input.cpp

```diff
@@ -769,8 +769,9 @@ std::string input_context::get_desc( const std::string &action_descriptor,
                                      const std::function<bool( const input_event & )> evt_filter ) const
 {
     if( action_descriptor == "ANY_INPUT" ) {
+        // \u00A0 is the non-breaking space
         //~ keybinding description for anykey
-        return string_format( pgettext( "keybinding", "[any] %s" ), text );
+        return string_format( pgettext( "keybinding", "[any]\u00A0%s" ), text );
```

Comment:
> translators might not be prepared to preserve it in their versions of these strings.

Transifex will show the non-breaking space using a special symbol. In the worst case when the translator is not paying attention, it will be translated into a normal space, which is just what it was before this change.

> Further, I'm not clear on the value of having it in the first place.

```
[a] press a key, [b]  | <- window boundary
press another key     |
```
compared with
```
[a] press a key      |
[b] press another key|
```
I believe the second one is more aesthetically pleasing.